examples/basic/basic-example.ts

```diff
@@ -141,7 +146,7 @@ client.on('message', async message => {
 			/*
 				The user is not in a voice channel
 			*/
-			message.reply('Join a voice channel then try again!');
+			void message.reply('Join a voice channel then try again!');
```

Comment:
```suggestion
			await message.reply('Join a voice channel then try again!');
```

Reply:
On the former one I can see that, but on this one its kinda useless, there is no catch or try/catch wrapping this, so voiding the promise ensures we don't care about it.